packages/editor/src/components/rich-text/index.native.js

```diff
@@ -0,0 +1,126 @@
+/**
+ * External dependencies
+ */
+import RCTAztecView from 'react-native-aztec';
+import { children } from '@wordpress/blocks';
+
+/**
+ * WordPress dependencies
+ */
+import { Component, RawHTML, renderToString } from '@wordpress/element';
+import { withInstanceId, compose } from '@wordpress/compose';
+
+export class RichText extends Component {
+	constructor() {
+		super( ...arguments );
+		this.onChange = this.onChange.bind( this );
+		this.onContentSizeChange = this.onContentSizeChange.bind( this );
+
+		this.lastContentSizeHeight = -1;
+		this.lastEventCount = 0;
+	}
+
+	/**
+	 * Handles any case where the content of the AztecRN instance has changed.
+	 */
+
+	onChange( event ) {
+		if ( !! this.currentTimer ) {
+			clearTimeout( this.currentTimer );
+		}
+		this.lastEventCount = event.nativeEvent.eventCount;
+		const newContent = event.nativeEvent.text.replace( /<p>/gi, '' ).replace( /<\/p>/gi, '<br>' );
+		this.lastContent = newContent;
+
+		this.currentTimer = setTimeout( function() {
+			this.props.onChange( {
+				content: this.lastContent,
+				eventCount: this.lastEventCount,
+			} );
+		}.bind( this ), 1000 );
+	}
+
+	/**
+	 * Handles any case where the content of the AztecRN instance has changed in size
+	 */
+
+	onContentSizeChange( event ) {
+		this.lastContentSizeHeight = event.nativeEvent.contentSize.height;
+		this.forceUpdate = true; // Set this to true and check it in shouldComponentUpdate to force re-render the component
+		this.props.onContentSizeChange( {
+			aztecHeight: this.lastContentSizeHeight,
+		}
+		);
+	}
+
+	shouldComponentUpdate( nextProps ) {
+		if ( !! this.forceUpdate ) {
```

Comment:
There is no need to convert values to boolean in JS. `this.forceUpdate` is all you need here and also it works the same in other cases below. JavaScript engine does type coercion itself.